src/parser/jobs/war/modules/PrimalProcs.tsx

```diff
@@ -0,0 +1,70 @@
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Procs as CoreProcs} from 'parser/core/modules/Procs'
+import {TieredSuggestion, SEVERITY} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SEVERITY_MISSED_PROCS = {
+	1: SEVERITY.MAJOR,
```

Comment:
_Note:_ From my armchair WAR position, I don't see anything wrong with flagging any drops of any of these procs as major, but I did want to check whether you poked a theorycrafter before setting the severity tiers (or are one yourself, in which case it's moot).

Reply:
Naw I'm not a warrior theorycrafter, I just figured "SEVERITY.MAJOR" was a reasonable translation of Mox from the Balance categorising these all as "big mistake" in a summary of the changes?
 
> Inner Release :ir: doesn't contain 3 Fell Cleaves => no Primal Wrath opened, big mistake
> Primal Wrath :wrath: opened (3x IR FC), but wasn't used (whoops dropped), big mistake
> Primal Ruination :ruination: opened, but wasn't used, big mistake

Primal Wrath is a 700 potency oGCD, Primal Ruination is 780p as well as guaranteed crit+DH. Happy to ping the good folks of the Balance to check those!

Reply:
Yep, let's call this major - https://discord.com/channels/441414116914233364/470049767477608459/1264640782535622668